tests/kernel/iop/messagebox/receive.c

```diff
@@ -0,0 +1,87 @@
+#include <common-iop.h>
+#include <thbase.h>
+#include <thmsgbx.h>
+#include "mbx-common.h"
+
+#define STACK_SIZE 0x800
+
+typedef struct
+{
```

Comment:
Nit: it's weird to see structs as the only thing using a different bracing style.  Doesn't really matter though.

-[Unknown]